packages/fx-core/templates/core/v3Migration/js.ts.app.yml

```diff
@@ -10,42 +10,77 @@ environmentFolderPath: ./ {{~environmentFolder}}
-      generateClientSecret: true # If the value is false, the action will not generate client secret for you
-      signInAudience: "AzureADMyOrg" # Authenticate users with a Microsoft work or school account in your organization's Azure AD tenant (for example, single tenant).
-    writeToEnvironmentFile: # Write the information of created resources into environment file for the specified environment variable(s).
+      # Note: when you run aadApp/update, the AAD app name will be updated
+      # based on the definition in manifest. If you don't want to change the
+      # name, make sure the name in AAD manifest is the same with the name
+      # defined here.
+      name: {{aadAppName}}
+      # If the value is false, the action will not generate client secret for you
+      generateClientSecret: true
+      # Authenticate users with a Microsoft work or school account in your
+      # organization's Azure AD tenant (for example, single tenant).
+      signInAudience: "AzureADMyOrg"
+    # Write the information of created resources into environment file for the
+    # specified environment variable(s).
+    writeToEnvironmentFile:
       clientId: AAD_APP_CLIENT_ID
-      clientSecret: SECRET_AAD_APP_CLIENT_SECRET # Environment variable that starts with `SECRET_` will be stored to the .env.{envName}.user environment file
+      # Environment variable that starts with `SECRET_` will be stored to the
+      # .env.{envName}.user environment file
+      clientSecret: SECRET_AAD_APP_CLIENT_SECRET
       objectId: AAD_APP_OBJECT_ID
       tenantId: AAD_APP_TENANT_ID
       authority: AAD_APP_OAUTH_AUTHORITY
       authorityHost: AAD_APP_OAUTH_AUTHORITY_HOST
 {{/if}}
-  - uses: teamsApp/create # Creates a Teams app
+  # Creates a Teams app
+  - uses: teamsApp/create
     with:
-      name: {{teamsAppName}} # Teams app name
+      # Teams app name
+      name: {{teamsAppName}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
     writeToEnvironmentFile:
-      # Write the information of created resources into environment file for the specified environment variable(s).
       teamsAppId: TEAMS_APP_ID
 {{#if activePlugins.fx-resource-bot}}
-  - uses: botAadApp/create # Creates a new AAD app for Bot Registration.
+  # Create or reuse an existing Azure Active Directory application for bot.
+  - uses: botAadApp/create
     with:
+      # The Azure Active Directory application's display name
       name: {{appName}}bt$\{{RESOURCE_SUFFIX}}
     writeToEnvironmentFile:
+      # The Azure Active Directory application's client id created for bot.
       botId: BOT_ID
+      # The Azure Active Directory application's client secret created for bot.
       botPassword: SECRET_BOT_PASSWORD
 {{/if}}
-  - uses: arm/deploy # Deploy given ARM templates parallelly.
+  - uses: arm/deploy  # Deploy given ARM templates parallelly.
```

Comment:
should we put the comment above `-uses: arm/deploy`?

Reply:
In the project templates, we place the comment in the same line and it has no yml lint warning. Here we just follow it.
There are some comments still placed in the end of line.